operators/multicluster-global-hub-operator/0.7.0-dev/multicluster-global-hub-operator.clusterserviceversion.yaml

```diff
@@ -30,10 +26,10 @@ metadata:
     categories: Integration & Delivery,OpenShift Optional
     certified: "false"
     containerImage: quay.io/stolostron/multicluster-global-hub-operator:latest
-    createdAt: "2023-05-10T20:00:36Z"
+    createdAt: "2023-06-12T02:01:09Z"
     description: Manages the installation and upgrade of the Multicluster Global Hub.
     operatorframework.io/suggested-namespace: open-cluster-management
-    operators.operatorframework.io/builder: operator-sdk-v1.23.0
+    operators.operatorframework.io/builder: operator-sdk-v1.28.0+git
```

Comment:
we should not change this

Reply:
@morvencao rollback to operator-sdk-v1.23.0